File: app/src/main/java/com/neome/java/api/meta/base/dto/DefnStudioPickLayoutSpreadsheetId.java
// neome.ai API - do not change
//

package com.neome.java.api.meta.base.dto;

import com.neome.java.api.meta.base.Types.EnumDefnLayoutGridKind;
import com.neome.java.api.meta.base.Types.MetaIdLayoutGrid;
import com.neome.java.api.meta.base.Types.MetaIdSpreadsheet;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class DefnStudioPickLayoutSpreadsheetId extends DefnFieldEditable
{
  @Nullable
  public MetaIdLayoutGrid[] excludeLayoutSpreadsheetIdSet;

  @Nullable
  public EnumDefnLayoutGridKind[] filterLayoutKindSet;

  @Nullable
  public Boolean showAlias;

  public MetaIdSpreadsheet spreadsheetId;
}

File: app/src/main/java/com/neome/api/meta/base/dto/DefnStudioPickLayoutSpreadsheetId.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.dto.DefnFieldEditable
import com.neome.api.meta.base.Types.EnumDefnLayoutGridKind
import com.neome.api.meta.base.Types.MetaIdLayoutGrid
import com.neome.api.meta.base.Types.MetaIdSpreadsheet

interface DefnStudioPickLayoutSpreadsheetId : DefnFieldEditable
{
  val excludeLayoutSpreadsheetIdSet: List<MetaIdLayoutGrid>?
  val filterLayoutKindSet: List<EnumDefnLayoutGridKind>?
  val showAlias: Boolean?
  val spreadsheetId: MetaIdSpreadsheet
}